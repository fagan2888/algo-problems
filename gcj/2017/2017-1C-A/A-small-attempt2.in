100
2 1
100 20
200 10
2 2
100 20
200 10
3 2
100 10
100 10
100 10
4 2
9 3
7 1
10 1
8 4
8 6
857788 123329
871774 296804
868008 796393
528547 713899
741262 589978
848331 141907
62634 987002
40005 674001
9 5
274536 60697
631098 26404
83421 199752
18837 884616
570024 29233
44919 370968
29016 574287
99876 166842
48546 343252
2 1
3065 341079
14941 5
8 5
3806 369115
3689 365360
12670 87
3699 302124
3799 345128
3190 335623
18531 20
3663 374272
3 1
189523 523914
144730 686062
851262 116643
2 1
786828 380165
652101 388732
10 6
20321 2373
20904 2253
23145 2577
24756 2827
23135 2599
26327 2137
229464 12
261741 25
292619 81
28861 2672
4 3
192190 341596
943738 383457
912605 244448
295811 405631
2 1
22630 2034
292858 5
10 7
21455 2530
27325 2234
21356 2452
25688 2840
213145 57
28506 2064
263973 93
260113 87
28874 2785
29017 2760
2 1
237369 23
28892 2529
9 6
131898 237434
363067 86257
157275 199123
68364 458093
364750 85859
581464 53859
36989 846659
37154 842899
664525 47127
3 2
191142 39468
373428 20202
633204 11914
4 3
119016 65527
72036 108262
94424 82593
533577 14616
2 1
403499 235501
217615 436663
1 1
1 1
9 6
128421 664953
321148 732631
424253 797795
626157 745803
993260 810231
727928 961354
654944 71932
942430 831050
860902 402503
6 1
864797 71859
328770 189018
75100 827476
199427 311610
400463 155179
67353 922653
7 4
180642 44892
28152 288057
13244 612306
27608 293733
587466 13804
92106 88044
9288 873103
9 2
23294 2688
208874 64
26383 2998
22260 2101
22010 2262
28449 2479
20477 2078
20496 2644
282203 89
9 6
832789 762109
436903 325734
680276 138830
935068 167097
922179 352590
502527 770873
510738 546385
569046 315443
114469 621460
10 5
3243 314760
14292 70
3629 340172
3864 347091
3601 375731
3471 329852
14513 99
14025 14
3207 383104
3363 380873
10 2
326390 811346
51944 310520
637444 285252
901344 960046
638722 334877
339912 307472
176117 751140
226199 482206
85225 583267
24446 357469
9 2
17169 71
3672 376777
3369 328526
3718 356845
3738 308353
3878 320815
3981 324498
11982 33
3958 398923
8 3
124285 831578
103238 93418
816842 292922
651636 148444
362905 270864
426984 23814
862215 971030
987370 744705
8 4
237071 99
29632 2824
20338 2712
24650 2408
22670 2388
28995 2513
20573 2181
264072 76
2 1
270788 132881
379327 94859
5 3
31731 664770
408680 393425
442569 242696
480730 865685
604548 778549
8 3
22478 2715
283847 16
25883 2154
24714 2576
27379 2127
270206 40
22879 2450
28771 2843
3 1
20835 2764
206311 40
27107 2419
9 6
20340 2558
26329 2955
29103 2153
23096 2031
28002 2438
204418 83
22526 2734
21916 2504
221247 11
8 5
3211 343920
3421 379915
3622 332569
11797 79
3532 360611
3625 367935
18995 68
3145 360173
3 2
834309 3432
369369 7752
874038 3276
10 5
107009 598108
509748 125558
745156 85892
475247 134673
170624 375111
124635 513523
101611 629882
468886 136500
71734 892226
67614 946593
7 1
139503 273758
56378 677393
560728 68108
143390 266337
129723 294397
386543 98799
420475 90826
7 4
130462 244188
411768 77367
265734 119884
380808 83657
66822 476748
130872 243423
368424 86469
7 1
106558 465736
131804 376528
393460 126132
508394 97617
280287 177061
353949 140212
105432 470710
9 5
11137 895752
80808 123453
44892 222222
95004 105006
27027 369112
33411 298584
24882 400932
97643 102168
19092 522522
6 4
87087 50184
50184 87087
4488 973791
18018 242556
150552 29029
7722 565964
7 5
344448 675543
342095 558567
275473 221166
509560 814752
906003 988635
50085 308681
603223 567925
7 2
470340 725503
660923 105814
631961 517365
214656 499854
785389 687392
109302 33894
156611 366690
5 3
19684 587466
47656 242649
23598 490028
338238 34188
913836 12654
5 3
24619 2656
252330 89
29109 2123
286160 96
28371 2003
10 6
112925 103541
742062 863150
231559 345441
700802 892665
814372 918544
247916 690724
578654 979047
567147 41977
237913 237805
105913 509640
10 6
283692 315404
228946 390824
302514 295780
137592 650311
408305 219144
143163 625005
675532 132455
474657 188510
256828 348395
162581 550357
9 5
28152 363909
142324 71982
359352 28509
34658 295596
14076 727818
56916 179998
467883 21896
513162 19964
71982 142324
8 5
3752 361982
13713 98
3785 398195
3804 344309
3778 374920
3436 325755
10484 58
3042 358176
3 2
291669 48024
31464 445179
94392 148393
8 5
318038 555818
54862 207093
477618 886124
526455 439374
165753 256928
421455 147848
230204 501846
24135 831596
9 6
3641 396165
17456 80
3180 375792
3883 311926
16883 46
3992 323525
3163 350509
3688 370300
3625 398452
10 4
11057 86
3397 305100
3933 347183
3500 380099
3095 353533
3836 339720
3142 375318
13501 78
3696 376670
13469 77
2 1
671666 65763
112730 391828
6 1
694830 136566
148217 640211
130727 725865
658260 144153
445270 213107
455944 208118
10 3
480359 134611
975450 66289
175248 368972
100226 645158
740844 87281
170540 379158
324289 199395
254150 254423
995192 64974
546373 118347
7 1
95857 252761
273903 88458
25661 944192
288766 83905
95284 254281
75264 321919
63049 384287
8 3
35705 426630
94035 161991
137931 110438
18420 826972
238516 63865
306835 49645
25907 587981
238516 63865
8 5
329189 45144
257796 57646
278388 53382
901098 16492
63612 233618
869364 17094
51414 289044
55924 265734
2 2
628488 79143
299367 166152
5 3
77256 284487
94392 232841
42021 523032
94829 231768
91908 239134
6 3
3341 353659
3386 303996
3209 344775
19440 60
13349 61
3804 341862
10 2
441086 319561
532379 906385
493602 351849
751245 466255
979356 650661
179264 31474
636309 750362
511474 177558
63962 718132
843568 943282
10 2
736662 936452
84933 717892
166217 75043
574805 234922
962847 445316
686729 736096
914530 521554
398216 814781
669999 665063
572992 425415
10 4
3091 389563
3224 357723
3173 305627
3757 361221
15439 30
16480 65
3598 361734
3519 391686
12058 79
3458 327953
9 5
88536 139113
269178 45756
87048 141491
24969 493272
17748 693966
282982 43524
420732 29274
960876 12818
139113 88536
3 1
168815 395722
322448 207177
94539 706627
10 10
1000000 1000000
1000000 1000000
1000000 1000000
1000000 1000000
1000000 1000000
1000000 1000000
1000000 1000000
1000000 1000000
1000000 1000000
1000000 1000000
8 2
3294 399785
3912 329593
3357 331314
3380 375420
16574 81
3136 302818
3780 302379
11223 57
8 5
627874 12852
720936 11193
99138 81396
583389 13832
107198 75276
14924 540702
360468 22386
194463 41496
3 1
836484 118791
608440 163314
246820 402588
8 5
223214 392840
278739 314586
803204 109172
224505 390581
173867 504336
989789 88592
138306 634010
234189 374430
2 1
893078 119340
48906 395336
6 1
26471 2950
22927 2900
259684 35
21877 2368
20960 2533
276562 86
9 6
23468 2556
20157 2759
212415 21
23914 2943
26388 2724
22022 2302
278714 4
28694 2325
24376 2561
3 1
202935 57
27323 2024
26036 2210
7 4
188969 698222
183658 309486
278839 690802
88439 75151
374720 712581
108882 228775
97724 140638
3 2
141372 49358
153846 45356
745416 9361
8 3
993385 411046
551887 250121
196889 47025
971743 256344
721657 818224
701086 814767
410665 323869
613482 535320
8 5
3105 385896
3121 378809
13790 61
3291 393500
3991 355586
3541 395073
3846 306927
18077 52
6 2
581117 862125
482598 636252
201998 249576
888379 23439
697209 527234
74301 130895
10 7
12297 32
3826 398291
3167 384930
15147 69
3644 353143
3841 387653
3395 316411
14511 8
3680 337295
3908 364135
10 5
12722 28
3483 357200
3144 397738
3862 323469
3142 360772
3115 339956
13849 72
3116 392536
12380 35
3131 343642
3 1
3210 366488
3286 372161
18564 72
4 1
3312 369934
3337 376103
12382 88
3629 388892
8 3
594741 146780
915706 95332
161587 540242
91421 954880
154833 563808
313965 278044
392097 222639
124553 700875
2 1
23958 2126
240226 73
5 3
27870 2445
258317 57
257920 64
20291 2949
23178 2322
3 2
30969 198968
10472 588411
302808 20349
6 3
25395 2213
21806 2070
22692 2666
26548 2132
222375 21
291900 80
2 1
160276 559289
344275 260375
2 1
258238 57
29914 2807
6 4
3324 317424
3259 338636
15710 20
3457 367966
11344 97
3174 307588
5 1
3035 90421
401661 199364
279527 238508
728630 30189
698733 561168
5 3
3177 308315
14090 26
3869 362813
3642 370830
12570 80
6 2
3589 349797
3071 370482
10184 51
12725 31
3667 383696
3636 308954
6 1
290491 23
29626 2831
23659 2356
271314 73
25850 2346
24665 2625
6 4
749398 32508
401964 60606
47601 511784
44289 550056
89096 273429
308826 78884
