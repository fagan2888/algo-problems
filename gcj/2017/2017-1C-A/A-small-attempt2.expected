Case #1: 138230.076758
Case #2: 150796.447372
Case #3: 43982.297150
Case #4: 625.176938
Case #5: 14896478258090.188000
Case #6: 1774747898335.202400
Case #7: 6597999578.898876
Case #8: 41214955090.729030
Case #9: 2900426721408.610400
Case #10: 3824409129790.927200
Case #11: 271180875780.777620
Case #12: 7227411257373.480000
Case #13: 269450433262.627350
Case #14: 221627036410.257800
Case #15: 177044331895.054320
Case #16: 2567932540734.968000
Case #17: 1354413714913.872800
Case #18: 1041428488116.750400
Case #19: 1108543260246.903000
Case #20: 9.424778
Case #21: 24711943942030.676000
Case #22: 2739973793812.418000
Case #23: 1288025802099.256300
Case #24: 250846616070.531740
Case #25: 15074592309160.734000
Case #26: 40498942054.714165
Case #27: 9653218405192.191000
Case #28: 18662915535.617170
Case #29: 14446656279763.926000
Case #30: 220558543674.837040
Case #31: 678125757584.440200
Case #32: 7730548848511.721000
Case #33: 254057836269.359650
Case #34: 133771319865.365660
Case #35: 155840241701.943300
Case #36: 39276260203.182594
Case #37: 2435977809838.711400
Case #38: 3755104416762.748000
Case #39: 1227721890505.016000
Case #40: 1333326243052.806400
Case #41: 1123811320222.250200
Case #42: 343357402433.596100
Case #43: 181046518720.833370
Case #44: 15630365039916.744000
Case #45: 7473981731636.673000
Case #46: 2841502443432.618700
Case #47: 258229055488.838650
Case #48: 19876797598453.570000
Case #49: 4806871055216.117000
Case #50: 1149140871210.008800
Case #51: 42210772675.285530
Case #52: 443276379117.957800
Case #53: 7211377187071.143000
Case #54: 50002837237.703460
Case #55: 33922224585.778057
Case #56: 1694816219313.114700
Case #57: 2112937943974.701700
Case #58: 4330298269153.134300
Case #59: 414198964401.231600
Case #60: 582905746219.479200
Case #61: 3017772081415.534700
Case #62: 1865976820264.548300
Case #63: 442532611731.910300
Case #64: 22592046971.006714
Case #65: 12016719774488.684000
Case #66: 9958898621067.795000
Case #67: 32975199629.644760
Case #68: 3287512636198.128400
Case #69: 746380623436.942900
Case #70: 65973445725385.660000
Case #71: 16423690192.837677
Case #72: 1886347577119.571000
Case #73: 2822529437320.854500
Case #74: 5832543124717.883000
Case #75: 3175359029032.477500
Case #76: 240438993037.776900
Case #77: 246132765347.926060
Case #78: 129451677762.870330
Case #79: 4515287488677.408000
Case #80: 1833296404193.545000
Case #81: 12964929545173.370000
Case #82: 40850378896.155556
Case #83: 6984627412606.628000
Case #84: 58213081654.966190
Case #85: 38377686255.545760
Case #86: 7717761824.656452
Case #87: 8908764383.656786
Case #88: 4279772775619.723000
Case #89: 181406882344.392850
Case #90: 210527803733.707950
Case #91: 365493030162.518400
Case #92: 268563786820.747620
Case #93: 935586655649.921900
Case #94: 209595449817.346600
Case #95: 27728012795.379690
Case #96: 3997491315247.112300
Case #97: 23507203859.447820
Case #98: 16770815016.149641
Case #99: 265145342103.099980
Case #100: 2376579668779.771500
